zedc/src/code/prepare.rs

```diff
@@ -81,6 +81,7 @@ fn code_cli_binary(dir: &Path) -> PathBuf {
 /// * `zip_path` - The path of the ZIP file to extract
 /// * `vsc_path` - The path where the archive should be extracted into
 ///
+#[allow(unused_variables)]
```

Comment:
Thanks for adding this, I've noticed Rust Analyzer reporting it even without the file being open 😀